apps/meteor/server/modules/listeners/listeners.module.ts

```diff
@@ -344,6 +344,10 @@ export class ListenersModule {
 			notifications.notifyUserInThisInstance(userId, 'voip.events', data);
 		});
 
+		service.onEvent('omnichannel.events', (userId, data): void => {
+			notifications.notifyUserInThisInstance(userId, 'omnichannel.events', data);
+		});
```

Comment:
I'm not sure informing every agent online (by user) about on inquiry being taken this is most performant approach.

If I'm not mistaken we already have the `roomData` that will be triggered and only who open the channel will receive

Reply:
agreed.. that would be much better to have some logic applied on on client side when data is received through the `room-data` stream